packages/libs/eda/src/lib/core/components/visualizations/implementations/ScatterplotVisualization.tsx

```diff
@@ -166,6 +166,24 @@ const modalPlotContainerStyles = {
   margin: 'auto',
 };
 
+// implement gradient color for slider opacity
+export const colorSpecProps: SliderWidgetProps['colorSpec'] = {
```

Comment:
since we're going to use this elsewhere, shall we give it a more descriptive name? Maybe `sliderOpacityColorSpecProps`? Or something with "gradient color" in it?

Reply:
I wonder do we have anywhere more general for this to live?

Reply:
I remember thinking the same thing, so I'll see what I can come up with!